contracts/service-registry/src/contract/query.rs

```diff
@@ -72,8 +73,14 @@ pub fn verifier(
     })
 }
 
-pub fn service(deps: Deps, service_name: String) -> Result<Service, ContractError> {
-    SERVICES
-        .may_load(deps.storage, &service_name)?
-        .ok_or(ContractError::ServiceNotFound)
+pub fn service(deps: Deps, service_name: String) -> error_stack::Result<Service, ContractError> {
+    state::default_service_params(deps.storage, &service_name)
+}
+
+pub fn service_params(
+    deps: Deps,
+    service_name: String,
+    chain_name: ChainName,
+) -> error_stack::Result<Service, ContractError> {
+    state::service(deps.storage, &service_name, &chain_name)
 }
```

Comment:
these names are confusing to me as well, i'd call it `service` and `service_with_overrides`. or `base_service` and `service_with_overrides`

Reply:
sounds good, I like the base_service/service_with_overrides combo. If some function is just `service` I would prefer it to be the most restrictive one, so in the future, we don't call `service` by accident when the intention was to return with overrides. However, the `QueryMsg` will still need to stay as just `Service` for the base service so we don't change the API and don't introduce a breaking change, what do you think?